pkg/test/resource_creator.go

```diff
@@ -34,7 +35,7 @@ func (ctx *TestCtx) GetNamespace() (string, error) {
 		return ctx.namespace, nil
 	}
 	// create namespace
-	ctx.namespace = ctx.GetID()
+	ctx.namespace = fmt.Sprintf("osdk-e2e-%s", uuid.New())
```

Comment:
I'd like to investigate adding unit tests around GetNamespace this weekend. Need to figure how to handle testing code using UUID. Attach a UUID on TestCtx? Maybe change GetID() to return this UUID on TestCtx instead of creating the UUID here.

Reply:
Taking a second look at this: I think assigning ctx.id in newTestCtx to the above string would make this all easier. So instead of changing resource_creator modify pkg/test/context.go's newTestCtx.